src/index.html

```diff
@@ -10,13 +10,66 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logotipo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li><a href="#">samsung</a></li>
+          <li><a href="#">smartphones</a></li>
+          <li>
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li><a href="#">gadgets</a></li>
+          <li><a href="#">tablets</a></li>
+          <li><a href="#">photo</a></li>
+          <li>
+            <a href="#">video</a>
+          </li>
```

Comment:
The 'href' attribute in this anchor tag is set to '#', which means it doesn't link to any specific page. Ensure to update it with the correct URL or section ID.